### PR TITLE
adding a new AQUFLUX case

### PR DESCRIPTION
**Pull request by GitPaean**
referring to discussion in https://github.com/OPM/opm-simulators/pull/4520 . 

This case provide more testing capacity and testing some different setup regarding the AQUFLUX. Especially, this case contains two aquifers and they are defined purely in the SCHEDULE section.  And it also revealed some bugs when developing PR  https://github.com/OPM/opm-simulators/pull/4520  when coming to parallel restart.

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this please

File: aquifers/AQUFLUX-02.DATA [hunk 1]
```diff
@@ -0,0 +1,307 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 Equinor
+-- Copyright (C) 2023 SINTEF ICT, Applied Mathematics
```
Comment (bska) [inline aquifers/AQUFLUX-02.DATA:7]: Are you really sure that you want to list SINTEF ICT here?

File: aquifers/AQUFLUX-02.DATA [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+-- Test case is testing analytical aquifer - constant flux
```
Comment (bska) [inline aquifers/AQUFLUX-02.DATA:9]: I think we should expand a little bit here to note how `-02` differs from `-01`.

File: aquifers/AQUFLUX-02.DATA [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+-- ---------------------------------------------------------------------------------------------------------------------------------
+-- ANALYTICAL AQUIFER INFLUX VARIABLES
+-- ---------------------------------------------------------------------------------------------------------------------------------
+--
+--       ANALYTICAL AQUIFERS(FIELD)
+--
+FAQR
+FAQT
+--
+--       ANALYTICAL AQUIFERS
```
Comment (bska) [inline aquifers/AQUFLUX-02.DATA:229]: We don't really support field-level aggregation of aquifer quantities yet so this test case won't write any non-trivial vectors to the summary file.  Maybe we could request `AAQR` and `AAQT` in addition to the field-level quantities?

**Review by bska — COMMENTED** [review]
Looks good for the most part.  I think we should amend the header a bit and probably request some additional summary output though.

File: aquifers/AQUFLUX-02.DATA (hunk 1 above)
Comment (GitPaean) [reply aquifers/AQUFLUX-02.DATA:7]: The case is based on `AQUFET-01.DATA`, which has a copyright claim from 2015. I will update it.

File: aquifers/AQUFLUX-02.DATA (hunk 3 above)
Comment (GitPaean) [reply aquifers/AQUFLUX-02.DATA:229]: Sure. I will do that.

Comment (GitPaean) [reply aquifers/AQUFLUX-02.DATA:229]: I think AAQR and AAQT are covered by `ALL` , maybe we should remove FAQR and FAQT, they are also covered by `ALL`.

Comment (bska) [reply aquifers/AQUFLUX-02.DATA:229]: > I think AAQR and AAQT are covered by `ALL`

Oh, yeah, you're right.  I didn't see `ALL` there.  That will absolutely cover the aquifer-related vectors so we don't have request those specifically.

**Review by bska — COMMENTED** [review]
Thanks for bringing `ALL` to my attention.  That will handle the aquifer-related vectors.

**Comment by GitPaean** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge into master.

**Pull request merged**